src/SimpleAzure.Storage.HybridQueue/HybridQueue.cs

```diff
@@ -7,21 +7,12 @@
 
 namespace WorldDomination.SimpleAzure.Storage.HybridQueues;
 
-public class HybridQueue : IHybridQueue
+public sealed class HybridQueue(QueueClient queueClient, BlobContainerClient blobContainerClient, ILogger<HybridQueue> logger) : IHybridQueue
 {
-    private readonly QueueClient _queueClient;
-    private readonly BlobContainerClient _blobContainerClient;
-    private readonly ILogger<HybridQueue> _logger;
+    private readonly QueueClient _queueClient = queueClient;
+    private readonly BlobContainerClient _blobContainerClient = blobContainerClient;
+    private readonly ILogger<HybridQueue> _logger = logger;

```

Comment:
I feel really weird about this.

But .. I think i'm warming to it though.

I get that it reduces some lines of code.
Also, I've got no other constructors here so again, starting to make some sense.

Also - `sealed` again...

Reply:
I'm going to roll with this. Something interesting, for a change.

Approved.